packages/vsx-registry/src/browser/vsx-extension.tsx

```diff
@@ -172,8 +172,15 @@ export class VSXExtension implements VSXExtensionData, TreeElement {
         return this.getData('version');
     }
 
+    /**
+     * Get an extension's average rating for display purposes.
+     * - `null` for published extensions that have a rating of 0.
+     * - `undefined` for built-in extensions (ratings should not be displayed).
+     *
+     * @returns the average rating of an extension if available.
+     */
     get averageRating(): number | undefined {
-        return this.getData('averageRating');
+        return !this.builtin ? this.getData('averageRating') : undefined;
```

Comment:
please revert, my comment was not for this accessor, but a logic inside `renderStars`